components/event-header/EventHeader.vue

```diff
@@ -35,9 +40,7 @@
       getNativeName: ISO6391.getNativeName,
     },
     props: {
-      name: String,
-      location: String,
-      allLocales: Array,
+      event: Object,
```

Comment:
Why remove the `props` validation?

Reply:
Very good question, I will bring it back :D